<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setup</class>
 <widget class="QDialog" name="Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Initial Setup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>230</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="drone1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 12pt &quot;Calibri&quot;;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Drone 1</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Select Available Drones: </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="drone2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 12pt &quot;Calibri&quot;;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Drone 2</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="drone3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 12pt &quot;Calibri&quot;;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Drone 3</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="drone4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 12pt &quot;Calibri&quot;;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Drone 4</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="drone5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 12pt &quot;Calibri&quot;;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Drone 5</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="drone6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 12pt &quot;Calibri&quot;;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Drone 6</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="drone7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 12pt &quot;Calibri&quot;;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Drone 7</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="drone8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 12pt &quot;Calibri&quot;;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Drone 8</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label2_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Sensor Update Frequency (second): </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sensorFrequencyLineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Plot X-axis Range (second):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xRangeLineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Distance Between Drones (meter): </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="droneDistanceLineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>drone1</tabstop>
  <tabstop>drone2</tabstop>
  <tabstop>drone3</tabstop>
  <tabstop>drone4</tabstop>
  <tabstop>drone5</tabstop>
  <tabstop>drone6</tabstop>
  <tabstop>drone7</tabstop>
  <tabstop>drone8</tabstop>
  <tabstop>sensorFrequencyLineEdit</tabstop>
  <tabstop>droneDistanceLineEdit</tabstop>
  <tabstop>xRangeLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Setup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Setup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
